### DISCUSSION TITLE
Myspace help

### DISCUSSION THREAD

**Discussion by Bu44man** [community/community#74789] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Can anybody tell me how to get my verification email from Myspace to come through to my email keeps saying it's sent but it didn't what am I doing on my space you asked I'm just trying to get something out of it if anybody can help please and thank you

**Comment by gitstua** [comment]
Hi @Bu44man ,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on [www.myspace.com](http://www.myspace.com/). It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
